src/algorithms/sdp.jl

```diff
@@ -100,7 +100,7 @@ end
 function sd_objective(objective0, sd_constraints, c::AbstractArray)
     function _objective(args)
         target = objective0(args)
-        barrier = sum(c .* -logdet.(sd_constraints.fs.(args)))
+        barrier = sum(c .* -logdet.(map(f -> f(args), sd_constraints.fs)))
```

Comment:
@mohamed82008 What makes this different? I found weird thing in my local version test yesterday before went to bed and I haven't pushed it, not sure if that's a bug. That if two MvNormal applied to one optimization then the first one is not well optimized, is this the reason?

Reply:
Yes, broadcasting a function calls it element-wise on the arguments. We need to call each function with all the arguments as input, hence the map.